travis/setup_dependencies_common.sh

```diff
@@ -9,6 +9,10 @@ conda config --add channels defaults
 
 shopt -s nocasematch
 
+LATEST_ASTROPY_STABLE=1.1
+LATEST_ASTROPY_LTS=1.0
+LATEST_NUMPY_STABLE=1.11
```

Comment:
Yes - I think it usually works ok because the ABI is less likely to change from say 1.11 to dev. But strictly speaking, dev/prerelease installations could break.